src/transformers/models/llava/processing_llava.py

```diff
@@ -134,3 +134,57 @@ def model_input_names(self):
         tokenizer_input_names = self.tokenizer.model_input_names
         image_processor_input_names = self.image_processor.model_input_names
         return list(dict.fromkeys(tokenizer_input_names + image_processor_input_names))
+
+    @property
+    def default_chat_template(self):
+        """
+        This default vicuna template formats inputs in the form of a chat history. For each message in the chat history:
+        * the template will output the role of the speaker followed by the content of the message.
+        * content is a list of strings and images.
```

Comment:
This is ambiguous for two things: 
* Not clear if "content" refers to input i.e. "content" section in messages, or the output (as this comes after referring to the template outputting "content of the message". 
* Is it a list of strings and images? If input it's a list of dicts; if output it's strings

Reply:
done, made more clear